proxies/live/apiproxy/targets/ig3.xml

```diff
@@ -1,62 +1,137 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <TargetEndpoint name="ig3">
-  <PreFlow>
-    <Request>
-      <Step>
-        <!-- Header NHSD-Session-URID must be present and not empty -->
-        <Name>RaiseFault.400BadRequest</Name>
-        <Condition>(request.header.NHSD-Session-URID is null) or (not request.header.NHSD-Session-URID ~~ ".+")</Condition>
-      </Step>
-      <Step>
-        <Name>OauthV2.VerifyAccessToken</Name>
-      </Step>
-      <Step>
-        <Name>Quota</Name>
-      </Step>
-      <Step>	
-        <Name>SpikeArrest</Name>	
-      </Step>
-      <Step>
-        <Name>AssignMessage.AddUserIdHeader</Name>
-      </Step>
-      <Step>
-        <Name>AssignMessage.AddIssuerHeader</Name>
-      </Step>
-      <Step>
-        <Name>KeyValueMapOperations.GetSecureVariables</Name>
-      </Step>
-      <Step>
-        <Name>AssignMessage.AddAsidHeader</Name>
-      </Step>
-    </Request>
-  </PreFlow>
-  <HTTPTargetConnection>
-    <Properties>
-      <!-- Whitelist the 'expected' error code responses so the responses pass unhindered to caller
+    <PreFlow>
```

Comment:
Probably best to put all this in the ProxyEndPoint PreFlow - the request-copy will (should?) be available here